diff --git a/common/lc_context.cpp b/common/lc_context.cpp
index 5f1bca4..08f18fa 100644
--- a/common/lc_context.cpp
+++ b/common/lc_context.cpp
@@ -11,6 +11,7 @@
 #include "lc_viewsphere.h"
 #include "lc_stringcache.h"
 #include "lc_partselectionwidget.h"
+#include <GL/gl.h>
 #include <QOpenGLFunctions_3_2_Core>
 
 #ifdef LC_OPENGLES
diff --git a/qt/lc_qpreferencesdialog.cpp b/qt/lc_qpreferencesdialog.cpp
index c6052ec..48e9531 100644
--- a/qt/lc_qpreferencesdialog.cpp
+++ b/qt/lc_qpreferencesdialog.cpp
@@ -1,3 +1,5 @@
+#include <GL/gl.h>
+
 #include "lc_global.h"
 #include "lc_qpreferencesdialog.h"
 #include "ui_lc_qpreferencesdialog.h"
