commit 1b0a47f692fa788166b065f3105085660436f5ef
Author: Leonardo Zide <leozide@gmail.com>
Date:   Tue Jan 12 18:29:11 2021 -0800

    Fixed aarch compile error.

diff --git a/common/lc_context.cpp b/common/lc_context.cpp
index 5f1bca4..ac4adfb 100644
--- a/common/lc_context.cpp
+++ b/common/lc_context.cpp
@@ -336,8 +336,10 @@ void lcContext::SetOffscreenContext()
 
 void lcContext::SetDefaultState()
 {
+#ifndef LC_OPENGLES
 	if (QSurfaceFormat::defaultFormat().samples() > 1)
 		glEnable(GL_LINE_SMOOTH);
+#endif
 
 	glEnable(GL_DEPTH_TEST);
 	glDepthFunc(GL_LEQUAL);
